encyclopedisque_importer.user.js

```diff
@@ -1,6 +1,6 @@
 ﻿// ==UserScript==
 // @name           Import Encyclopedisque releases to MusicBrainz
-// @version        2018.2.18.1
+// @version        2020.09.13.1
```

Comment:
Version numbers with a leading zero may cause problems

Reply:
Thanks for the reminder! 😉